platform/spire/src/access.py

```diff
@@ -87,13 +90,23 @@ def renew_ssh_cert() -> str:
 
 
 def refresh_cert(key_path, cert_path, ca_path, variant, ca_key_name, ca_cert_name):
-    if configuration.get_config().is_kerberos_enabled():
-        print("rotating", variant, "certs via keyreq")
-        if ca_path is None:
-            call_keyreq(variant + "-cert", key_path, cert_path)
-        else:
-            call_keyreq(variant + "-cert", key_path, cert_path, ca_path)
-    else:
+    errs = []
+
+    try:
+        if configuration.get_config().is_kerberos_enabled():
+            print("rotating", variant, "certs via keyreq")
+            if ca_path is None:
+                call_keyreq(variant + "-cert", key_path, cert_path)
+            else:
+                call_keyreq(variant + "-cert", key_path, cert_path, ca_path)
+            return
+    except Exception as e:
+        print("[keyreq failed]")
+        if os.environ.get('SPIRE_DEBUG'):
+            traceback.print_exc()
```

Comment:
Could we have `[keyreq failed]` include a note that you can set SPIRE_DEBUG to get more info?